bin/snapshot-backups/site-snapshot-delete

```diff
@@ -0,0 +1,79 @@
+#!/usr/bin/env bash
+
+# we might be running in circleci
+if [ -f /home/circleci/project/env.local ]; then
+  . /home/circleci/project/env.local
+fi
+# we might be running from a local dev machine
+SCRIPT_DIR="$(dirname "$0")"
+if [ -f $SCRIPT_DIR/env.local ]; then
+  . $SCRIPT_DIR/env.local
+fi
+if [ -f ./env.local ]; then
+  . ./env.local
+fi
+
+# just testing?
+if [ x$1 == x"--dryrun" ]; then
+  export echo=echo
+  shift
+fi
+
+DEPLOY_TAG=${1}
+shift
+
+if [ -z "$DEPLOY_TAG" ]; then
+  echo
+  echo "Usage: $0 [--dryrun] <tag> [--download]"
+  echo "This will remove a tagged static site snapshot. Use static-site-list to see available tags."
+  echo "  --dryrun simply prints the commands that would have run"
+  echo "  --download downloads and creates a zip file of the snapshot, prior to deletion"
+  echo
+  exit 1
+fi
+
+$echo source bin/cloudgov/get-s3-access storage >/dev/null 2>&1
```

Comment:
I want to point out something about using $echo here; I'm not sure we want to change it but ... 
In --dryrun mode, this skips running get-s3-access, which may be what you mean, but it results in the bucket name not being defined and in me squinting at three slashes in output like: 

`aws s3 rm --only-show-errors --recursive s3:///web-backup/USAGOV-784.dev.
`
I think we might actually want to run get-s3-access during the dryrun to verify it works? But maybe not. I could go either way.